extensions/2.0/Vendor/EXT_lights_image_based/README.md

```diff
@@ -0,0 +1,127 @@
+
+The entire mip chain of images should not be provided. Instead, the lowest-resolution mip should have sufficient size to represent the maximally-blurred radiance map (say, 16x16) corresponding to roughness=1. The `specularImageSize` value defines the largest dimension of mip 0 and, taken together with the number of defined mips, should give the loading runtime the information it needs to generate the remainder of the mip chain and sample the appropriate mip level in the shader.
+
+*(say something about tooling for generating these mips)*
+
+Cube faces are defined in the following order and must be authored with the Y-axis inverted: 
+1. Positive X
+1. Negative X
+1. Positive Y
+1. Negative Y
+1. Positive Z
+1. Negative Z
+
+## Irradiance Coefficients
+
+This extension defines spherical harmonic coefficients up to l=2 for irradiance lighting. This takes the form of a 9x3 array.
+https://cseweb.ucsd.edu/~ravir/papers/envmap/envmap.pdf
+http://graphics.stanford.edu/papers/envmap/
+
+Generating SH Coefficients
+http://graphics.stanford.edu/papers/envmap/prefilter.c
+
+Using SH Coefficients
+http://graphics.stanford.edu/papers/envmap/envmaprender.in
+
+*(this section needs more about how the coefficients are used as well as a brief blurb about what SH are)*
+*(also, we should probably just put some shader code in here for using the coefficients)*
+
+## HDR Images
+
+This extension also provides a way to reference HDR data which, up to this point, has not been possible within glTF. The individual images referenced by this extension can either be:
+1. JPEG's - assumed to be regular, LDR, RGB images
+2. PNG's - assumed to be HDR images packed in RGBD format.
+
+### RGBD
+RGBD is a way to pack HDR data so that the alpha channel contains a value that, when the RGB channels are divided by it, can produce values greater than 1. Runtimes can choose to unpack this data to float textures at load time (recommended) or keep it packed on the GPU (and unpack in the shader) to save memory.
+
+RGBD has several advantages over similar packing schemes like RGBM and RGBE.
+
+1. Simple to pack and unpack. Cheaper than RGBE.
+2. Provides better range than RGBM.
+3. Interpolates better than RGBE. Important if the data is kept packed during rendering and filtering is used. Note that the interpolation isn't perfect so visual artifacts may still appear. This is why unpacking to float at load-time is recommended.
+4. Packed PNG doesn't appear as garbled as a one packed with RGBE.
+
+### Future HDR Image Support
+Because this extension just references regular textures, any future support added by glTF for HDR formats will automatically be supported by this extension. e.g. CTTF formats.
```

Comment:
This statement contrasts oddly with the "exactly one way..." statement in the Overview above.